src/cascadia/TerminalApp/ActionPaletteItem.cpp

```diff
@@ -24,27 +24,5 @@ namespace winrt::TerminalApp::implementation
         Name(command.Name());
         KeyChordText(command.KeyChordText());
         Icon(command.IconPath());
-
-        _commandChangedRevoker = command.PropertyChanged(winrt::auto_revoke, [weakThis{ get_weak() }](auto& sender, auto& e) {
-            auto item{ weakThis.get() };
-            auto senderCommand{ sender.try_as<Microsoft::Terminal::Settings::Model::Command>() };
-
-            if (item && senderCommand)
-            {
-                auto changedProperty = e.PropertyName();
-                if (changedProperty == L"Name")
-                {
-                    item->Name(senderCommand.Name());
-                }
-                else if (changedProperty == L"KeyChordText")
-                {
-                    item->KeyChordText(senderCommand.KeyChordText());
-                }
-                else if (changedProperty == L"IconPath")
-                {
-                    item->Icon(senderCommand.IconPath());
-                }
-            }
-        });

```

Comment:
ah. Don't we still need this? I guess not because a settings reload will reload the entire command palette? We never really need to update the commands on the fly?

Reply:
> We never really need to update the commands on the fly

We sure don't! When would we ever set a property on a Command at runtime, that doesn't just involve blowing away all the settings and rebuilding them?

Reply:
And yet... didn't we implement this for some specific reason?